### DISCUSSION TITLE
X and Y coordinates

### DISCUSSION THREAD

**Discussion by bruzie** [theOehrly/Fast-F1#116] [Q&A] [answered]
Any idea on how the X and Y coordinates work in terms of the source? I understand how to use them in a scatter plot (e.g. gear selection), but in my experience they do not match with the cars' actual position, nor do they match with the centre line of the track.

For example, I've created a plot using the Brazil GP data showing turn 4, and placed a screen grab from Google behind it and adjusted the plot ratio so they line up with each other. 

[This chart shows the X/Y for VER for laps 46 to 50](https://i.imgur.com/ZoSms5B.png). As you can see, all laps follow the exact same line. If it was actual position, I would expect [L48 to be running off the track](https://youtu.be/uhrAgoHCoyo) but this decidedly not the case. It appears that X/Y follow the ideal race line.

I know the official F1 telemetry is un-documented, and I know it's not as granular as what the teams get (and Jolyon gets access to), but is there any explanation for this?

Edit: I've just done a quick chart for Barcelona (quickest race laps in 2020 and 2021) [that shows the track changes](https://i.imgur.com/85oA39Z.png) between the two years.

**Comment by theOehrly** [comment, marked as answer]
I guess the coordinates are only intended for driver tracker graphics and scatter plots and the like, as you noted. I would call it something like a "normalized track position".

All coordinates show the same line. Minor deviations are likely due to quantization inaccuracies. The line tends to approximately follow the ideal racing line around a circuit.

It's certainly not possible to analyze drivers lines through a corner. 

As to why this is how it is, I'd say it comes down to what the intended use for this data is. The official app uses it to show the driver tracker. This "normalized track position" is probably better suited for this.

**Reply by bruzie** [reply]
That seems to make perfect sense - it's not really supposed to be used for driver analysis, but to provide tracking data for live visualisations.
